lib/venue.rb

```diff
@@ -1,2 +1,40 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+
+    @patrons = []
+  end
+
+
+  def add_patron(patron)
+    patrons << patron
+  end
+
+
+  def yell_at_patrons
+  upcase_patrons = []
+    patrons.each do |patron|
+      upcase_patrons << patron.upcase
+    end
+  upcase_patrons
+  end
+
+  def over_capacity
+    if patrons.length > capacity
+       true
+    else
+      false
+    end
+  end
+
+  def kick_out
+    while patrons.length > capacity
```

Comment:
Nice use of the `while` loop here. Is there a method that is already doing `patrons.length > capacity`? So you could write this like:
```
while over_capacity? == true patrons.pop
```